Gems/PhysX/Code/Source/Shape.cpp

```diff
@@ -129,6 +129,18 @@ namespace PhysX
         return nullptr;
     }
 
+    void Shape::SetMaterialSelection(const Physics::MaterialSelection& materialSelection)
+    {
+        if (m_pxShape)
+        {
+            AZStd::vector<physx::PxMaterial*> materials;
+            MaterialManagerRequestsBus::Broadcast(&MaterialManagerRequestsBus::Events::GetPxMaterials, materialSelection, materials);
+            m_pxShape->setMaterials(materials.begin(), aznumeric_cast<physx::PxU16>(materials.size()));
```

Comment:
nit: Should this be `materials.data()` instead of `.begin()` technically (if this is a pointer and a size and not `begin`/`end` iterators?

Reply:
good point, removed this function now!